cli/tools/installer.rs

```diff
@@ -139,6 +139,47 @@ pub fn infer_name_from_url(url: &Url) -> Option<String> {
   Some(stem)
 }
 
+pub fn uninstall(name: String, root: Option<PathBuf>) -> Result<(), AnyError> {
+  let root = if let Some(root) = root {
+    canonicalize_path(&root)?
+  } else {
+    get_installer_root()?
+  };
+  let installation_dir = root.join("bin");
+
+  // ensure directory exists
+  if let Ok(metadata) = fs::metadata(&installation_dir) {
+    if !metadata.is_dir() {
+      return Err(generic_error("Installation path is not a directory"));
+    }
+  }
+
+  let mut file_path = installation_dir.join(&name);
+
+  let mut removed = false;
+
+  if file_path.exists() {
+    fs::remove_file(&file_path)?;
+    println!("deleted {}", file_path.to_string_lossy());
+    removed = true
+  };
+
+  if cfg!(windows) {
```

Comment:
Minor point, but could this copy what's done for install and instead...

```rust
if cfg!(windows) {
 file_path = file_path.with_extension("cmd");
}
```

...could be done above? It also looks like the logic for getting the installation dir and file path could be shared between `install` and `uninstall`.

Reply:
Nevermind. Just saw and remembered that this is necessary because it outputs a shell script on windows.